pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: news_post
+title: "2017 Fukuoka Ruby Award Competition - Entradas serão avaliadas pelo Matz"
+author: "Fukuoka Ruby"
+translator: "Filipe Rocha"
+date: 2016-10-20 00:00:00 +0000
+lang: pt
+---
+
+Caros entusiastas de Ruby,
+
+O Governo de Fukuoka, Japão, juntamente com "Matz" Matsumoto gostaria de
+vos convidá-lo a entrar no seguinte concurso Ruby. Se já desenvolveu um
+programa interessante em Ruby, por favor sinta-se encorajado a participar.
+
+2017 Fukuoka Ruby Award Competition - Grande Prémio - 1 milhão de ienes!
+
+Data limite para inscrição: 27 de dezembro de 2016
+
+![Fukuoka Ruby Award](http://www.digitalfukuoka.jp/javascripts/kcfinder/upload/images/fukuokarubyaward2017.png)
+
+Matz e um grupo de palestrantes irá selecionar os vencedores do Fukuoka Ruby Award.
+O prémio para esta competição é de 1 milhão de ienes (≈8480€).
+Entre muitos, alguns dos vencedores anteriores do prémio são Rhomobile (EUA) e APEC Climate Center (Coreia).
```

Comment:
In pt-br we write `prêmio` with `ê` instead of `é`. I know that this differs in pt-pt, just wanted to point that out. Has this changed for you after the language agreement?